subs-ui.js

```diff
@@ -47,7 +47,16 @@ function buildMenuButtonContainer() {
     return menuButtonContainer;
 }
 
+function deleteOldButton(ID) {
+    const oldButton = document.querySelector(`#${ID}`);
+    if (oldButton) {
+        oldButton.remove();
+    }
+}
+
 function addSettingsButton() {
+    deleteOldButton(SETTINGS_BTN);
```

Comment:
good idea, this should avoid the issue that occasionaly pops up where the top ui buttons are there twice